node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
this file (and the directory for that matter) shouldn't be here either (but i get why it is in this instance). just remove this file from the commit.

Reply:
This was deleted